enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,256 @@
+
+#### Local volumes
+
+A `localVolume` mounts a path from the node's file system tree or a separate disk as a volume into a Pod.
+
+Utilising a `local StorageClass` with no provisioner, guarantees a behaviour similar to dynamic provisioning.
+Upon creating a `PVC` with the `local StorageClass`, a `PV` will be allocated using `localVolume` and bound to the `PVC`.
+Since the `PV` is actually a path on the node's file system, the etcd backup will be always available regardless of the Pod's status, as long as the node itself is still accessible.
+using `localVolume` and the node affinity within the `PV` manifest forces the backup pod to be scheduled on a specific node, where the volume is attached.
+Had the Pod been replaced, the new Pod is guaranteed to be scheduled on the same node, where the backup exists.
+This deterministic scheduling behaviour is a property of `localVolume`, which is not the case with `hostPath` volumes.
+`localVolume` is handled by the PV controller and the scheduler in different manner, in fact it was created to resolve issues with `hostPath`
+
+* Pros
+  - It supports all Openshift variants, including `SNO` and `BM`.
+  - The Backup Pod will be scheduled always to this master node where the backup data exists.
+  - Using a separate disk for backups (e.g. SSD) is possible, since local volume allows mounting it into a Pod.
+  - Having the deterministic Pod scheduling is vital, since the restoration needs the recent backup in place to work.
+* Cons
+  - If the master node where the `PV` is mounted became unhealthy/unavailable/unreachable, then the backups are no longer accessible, also taking new backups is no longer possible.
+
+#### Host Path volumes
+
+A `hostPath` volume mounts a path from the node's file system as a volume into a Pod.
+
+* Pros
+  - It supports all Openshift variants, including `SNO` and `BM`.
+* Cons
+  - `hostPath` could have security impact as it exposes the node's filesystem.
+  - No scheduling guarantees for the pod using `hostpath` as is the case with `localVolume`. The pod could be scheduled on a different node from where the hostPath volume exist.
+
+### Hybrid approach
+
+As shown above, dynamic provisioning is the best storage solution to be used, but it is not viable among all Openshift variants.
+However, a hybrid solution is possible, in which dynamic provisioning is being used in cloud based Openshift, while utilising local storage for BM and SNO.
+Relying on `infrastructure` resource type, we can create the storage programmatically according to the underlying infrastructure and Openshift variant.
+
+### StatefulSet approach
+
+A `StatefulSet` could be deployed among all master nodes, where each backup pod has its own `PV`. This approach has the pros of spreading the backups among all master nodes.
+The complexity come from the fact that the backups are being triggered by a `CronJob` which spawn a `Job` to take the actual backup, by deploying a Pod.
```

Comment:
CEO could add the nodeName to the podSpec in the Job when it creates it and handle this